src/validation/custom.validation.ts

```diff
@@ -295,3 +295,35 @@ const checkCorporatesAddress = (trust: trustType.Trust, addressMaxLength: number
     }
   }
 };
+
+export const validateEmail = (email: string, maxLength: number) => {
+  const isEmailPresent = checkEmailIsPresent(email);
```

Comment:
You do not need to check it with ifs something like the following will do
```
export const validateEmail = (email: string, maxLength: number) => {
    checkEmailIsPresent(email);
    checkIsWithinLengthLimit(email, maxLength);
    checkCorrectIsFormat(email);
    
    return true;
}
```

Reply:
This seems to work so removed the conditions